src/package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Create UI project directly in the root folder. i.e. `Morph/src/package.json` should be in `Morph/package.json`.